src/components/Extensibility/components-form/MultiCheckbox.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+
+import { ResourceForm } from 'shared/ResourceForm';
+import * as Inputs from 'shared/ResourceForm/inputs';
+import {
+  useGetTranslation,
+  getPropsFromSchema,
+} from 'components/Extensibility/helpers';
+
+function getValue(storeKeys, resource) {
+  let value = resource;
+  const keys = storeKeys.toJS();
+  keys.forEach(key => (value = value?.[key]));
+  return value;
+}
+
+export function MultiCheckbox({
+  onChange,
+  // value,  //<-- doesn't work
+  schema,
+  storeKeys,
+  required,
+  resource,
+  compact,
+  placeholder,
+}) {
+  const { tFromStoreKeys, t: tExt, exists } = useGetTranslation();
+  const schemaPlaceholder = schema.get('placeholder');
+  const readOnly = schema.get('readOnly') ?? false;
+  const value = getValue(storeKeys, resource);
```

Comment:
We already have a workaround for this.
```suggestion
  value = getObjectValueWorkaround(schema, resource, storeKeys, value);
```
(see for example `src/components/Extensibility/components-form/ResourceRefRenderer.js`)